test/proxy.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * The talkback proxy for e2e tests. When making a request to the API during testing,
+ * it tries to use the responses it previously saved in `/test/tapes` folder. If no
+ * response is found there, it makes an actual request, and saves the response for
+ * future use.
+ * This makes it possible for the e2e tests to run without internet, and makes the
+ * tests less flaky due to changes in the API or API data.
+ */
+const talkback = require('talkback')
+const host = 'https://api.openverse.engineering/v1'
```

Comment:
This comment https://github.com/WordPress/openverse-frontend/pull/881#discussion_r810079110 got resolved with this changing or explanation why we shouldn't. Was that just an oversight or was there a reason we need to keep the `v1` prefix for the routes?

Reply:
The API service uses the `host` URL to construct the request URL, so for search it would use `${host}/search/?=galah`. If we don't add `v1`, then all the requests to the API go to the wrong URL (without `v1`), and the tests fail.

I think it's best to create an issue about it and either change the API service, adding `/v1` to each request function individually, or change the Analytics server (why isn't it using `/v1/` ?).

Reply:
:scream: I totally missed that that would be a problem when talkback actually has to forward the request :grimacing: 

I assumed the analytics endpoints aren't under `v1` because they're technically not part of the open API of Openverse, they're just internal metrics gathering tools? :shrug: 

Agreed that the frontend should probably be making the requests itself.

I wonder if a stop-gap is to have _two_ talkback servers running? Or is that overkill and we should just wait and fix it for real? :thinking:

Reply:
> I assumed the analytics endpoints aren't under v1 because they're technically not part of the open API of Openverse, they're just internal metrics gathering tools? 🤷

that's right. it's a one line NGINX config change in the infra if we want to change this, btw!

Reply:
Issue opened: #915